assets/reader-activation/index.js

```diff
@@ -383,23 +383,28 @@ function attachAuthCookiesListener() {
  * Set the reader as newsletter subscriber once a newsletter form is submitted.
  */
 function attachNewsletterFormListener() {
-	const forms = document.querySelectorAll(
-		'.newspack-newsletters-subscribe,.newspack-subscribe-form,.mc4wp-form'
-	);
-	if ( ! forms.length ) {
+	const newspackForms = [ '.newspack-newsletters-subscribe', '.newspack-subscribe-form' ];
+	const thirdPartyForms = [ '.mc4wp-form' ];
+
+	if ( ! newspackForms.length && ! thirdPartyForms.length ) {
```

Comment:
Isn't this always going to return true? It looks like we are creating two arrays here with hard-coded entries, and then checking for lengths.

Reply:
You're right—this is leftover from before I did the hardcoded arrays thing. Updated in 5ec8bba